scene/2d/parallax_2d.h

```diff
@@ -0,0 +1,97 @@
+/**************************************************************************/
+/*  parallax_2d.h                                                         */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef PARALLAX_2D_H
+#define PARALLAX_2D_H
+
+#include "scene/2d/node_2d.h"
+
+class Parallax2D : public Node2D {
+	GDCLASS(Parallax2D, Node2D);
+
+	String group_name;
+	Size2 scroll_scale = Size2(1, 1);
+	Point2 scroll_offset;
+	Point2 screen_offset;
+	Vector2 repeat_size;
+	int repeat_times = 1;
+	Point2 limit_begin = Point2(-10000000, -10000000);
+	Point2 limit_end = Point2(10000000, 10000000);
```

Comment:
I would define the `10000000` as `PARALLAX_2D_DEFAULT_LIMIT`.

Reply:
Good point. I used `Camera2D` as an example. Perhaps I should make a PR to add a constant for `Camera2D` as well.